pytket/binders/circuit/main.cpp

```diff
@@ -81,6 +81,15 @@ PYBIND11_MODULE(circuit, m) {
             const auto &gate = static_cast<const Gate &>(*op);
             return gate.get_unitary();
           })
+      .def(
+          "is_clifford",
+          [](const Op &op) {
+            if (op.get_desc().is_gate() && !op.get_desc().is_clifford_gate() &&
```

Comment:
This method will return `true` if the operation is not a gate. Is that what we want?